### DISCUSSION TITLE
Error 502 - Bad Gateway

### DISCUSSION THREAD

**Discussion by phaticusthiccy** [lowlighter/metrics#867] [Q&A]
Sometimes the site downs. You can search for a solution.

![Ekran Görüntüsü (181)_LI](https://user-images.githubusercontent.com/74794033/153427971-6f050281-b95f-4105-ab40-afe1d12a6387.jpg)

**Comment by lowlighter** [comment]
The server hosting it had a `/var` full making it a ro system
I think adding a `ExecStartPre=/usr/bin/docker system prune -af` to the systemd service should fix the issue, I noticed that dangling images of previous launches were not properly purged so it was filling up disk space.

The website is now up, I restarted it manually 👍 

However I can't guarantee you that the website will have a 100% uptime.
The VPS I rent is quite cheap so it doesn't have a lot of resources on it but normally if the service crashes it should restart automatically.
But don't hesitate to ping me again if it's down again, I'll look into it 🙂
